lib/order_list.rb

```diff
@@ -0,0 +1,51 @@
+require_relative 'dish_list'
+require 'twilio-ruby'
+
+class OrderList
+
+  extend Forwardable
+
+  def_delegator :@dish_list, :price_list
+
+  attr_reader :client
+
+  def initialize(dish_list)
+    @dish_list = dish_list
+    account_sid = 'AC052294c8bfb6a8bccf164140b4116340'
+    auth_token = '9db8e3404e5865a71a79d28c3ef11e71'
+    @client = Twilio::REST::Client.new(account_sid, auth_token)
+  end
+
+  def place_order(amount_list,total=nil)
+    fail 'An ordered dish is unavailable' unless ordered_dishes_available?(amount_list)
+    fail 'Total does not match sum of dishes in order' unless total == nil || total == dish_sum(amount_list)
```

Comment:
Metrics/LineLength: Line is too long. [108/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/NilComparison: Prefer the use of the `nil?` predicate. (https://github.com/bbatsov/ruby-style-guide#predicate-methods)